index.html

```diff
@@ -266,5 +266,32 @@ <h3 class="skillsTitle">Skills</h3>
   </div>
 </section>
 
+            <!-- This is the contact form -->
+
+<section class="contact" id="contactForm">
+    <h2 class="contactHeader">Contact me</h2>
+    <p class="contactDescription">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+    <form action="https://formspree.io/f/mgedkave" class="contactForm" method="post">
+      <ul class="contactField">
+        <li>
+          <label for="name"></label>
+          <input class = "contactName" type="text" id="name" name="name" placeholder="Enter your name here ..." maxlength="30" required />
+        </li>
+        <li>
+          <label for="email"></label>
+          <input class="emailAddress" type="email" id="email" name="email" placeholder="Enter your e-mail here ..." required />
+        </li>
+        <li>
+          <label for="message"></label>
+          <textarea class="contactMessage" id="message" placeholder="Type your message here ..." maxlength="500" required></textarea>
+        </li>
```

Comment:
Ah yes.. when I first saw it I was thinking that someone just wrote it there as an example to see which one should be the name field and which one is the e-mail field.. Didn't really knew that that was required.

Reply:
I have never seen forms with a real name / e-mail address as a placeholder so that's why it made more sense to use "Type your name ..." and so on @Kingstalux